spawn.lua

```diff
@@ -1,66 +1,45 @@
+function SendToWorldSpawn( Player )
+
+	local World = Player:GetWorld()
+	Player:TeleportToCoords(World:GetSpawnX(), World:GetSpawnY(), World:GetSpawnZ())
```

Comment:
Okay, I now see that there's some code specifically for players that might take care of the falling through. Still, I consider it good practice to teleport to chunks that actually exist.